core/croquet/src/main/java/org/lgna/croquet/views/DragComponent.java

```diff
@@ -302,50 +292,42 @@ private void handleMouseReleased(MouseEvent e) {
       ConsistentMouseDragEventQueue eventQueue = ConsistentMouseDragEventQueue.getInstance();
       eventQueue.popClickAndClackMouseFocusComponentButAllowForPotentialFollowUpClickEvent();
     }
-    if (isActuallyPotentiallyDraggable()) {
-      if (isClack || MouseEventUtilities.isQuoteLeftUnquoteMouseButton(e)) {
-        if (this.isWithinClickThreshold) {
-          if (this.isClickAndClackAppropriate()) {
-            ConsistentMouseDragEventQueue eventQueue = ConsistentMouseDragEventQueue.getInstance();
-            if (eventQueue.isClickAndClackSupported()) {
-              Component peekComponent = eventQueue.peekClickAndClackComponent();
-              Component awtComponent = this.getAwtComponent();
-              if (awtComponent == peekComponent) {
-                //pass
-              } else {
-                eventQueue.pushClickAndClackMouseFocusComponent(awtComponent);
-              }
-            }
-          } else {
-            this.handleLeftMouseButtonQuoteClickedUnquote(e);
-          }
-        } else {
-          if (this.isDueQuoteExitedUnquote) {
-            this.handleMouseQuoteExitedUnquote();
-          }
-          //        this.setActive( this.getAwtComponent().contains( e.getPoint() ) );
-          JLayeredPane layeredPane = getLayeredPane();
-          Rectangle bounds = this.dragProxy.getBounds();
-          layeredPane.remove(this.dragProxy);
-          layeredPane.repaint(bounds);
-          if (this.dragStep != null) {
-            this.dragStep.handleMouseReleased(e);
-            this.dragStep = null;
+    if (!isActuallyPotentiallyDraggable() || (!isClack && !MouseEventUtilities.isQuoteLeftUnquoteMouseButton(e))) {
+      return;
+    }
+    if (this.isWithinClickThreshold) {
+      if (this.isClickAndClackAppropriate()) {
```

Comment:
maybe not right here, but can we have the comment somewhere about what the heck a clack is?!

Reply:
I can only add that comment after I learn what it is

Reply:
hahaha, i accept